application-mocca-calendar-ui/src/main/resources/MoccaCalendar/WebHome.xml

```diff
@@ -70,6 +72,23 @@
 })
 #set ($columns = ['doc.title', 'doc.creator', '_actions'])
 #livetable('moccacalendar' $columns $columnsProperties $options)
+(% style="border-color:gray" %)
```

Comment:
You should opt for a color defined as a LESS variable. You should also opt for a style sheet rather than inline styling.

Reply:
I've removed the CSS to have the default color for the separator.